aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -103,16 +103,18 @@ impl Deref for MoveVmExt {
     }
 }
 
-pub fn verifier_config(treat_friend_as_private: bool) -> VerifierConfig {
+pub fn verifier_config(_treat_friend_as_private: bool) -> VerifierConfig {
     VerifierConfig {
         max_loop_depth: Some(5),
-        treat_friend_as_private,
         max_generic_instantiation_length: Some(32),
         max_function_parameters: Some(128),
         max_basic_blocks: Some(1024),
         max_value_stack_size: 1024,
         max_type_nodes: Some(256),
-        max_dependency_depth: 256,
+        max_dependency_depth: Some(256),
```

Comment:
Is this intended?

Reply:
Yes, because it caused failure in the CI